gapis/api/gles/api/egl.api

```diff
@@ -357,15 +357,15 @@ cmd EGLBoolean eglSurfaceAttrib(EGLDisplay display,
   return ? // TODO
 }
 
-@no_replay
 @frame_start
+@custom
 ///http://www.khronos.org/registry/egl/sdk/docs/man/html/eglSwapBuffers.xhtml
 cmd EGLBoolean eglSwapBuffers(EGLDisplay display, void* surface) {
   return ?
 }
 
-@no_replay
 @frame_start
+@no_replay
```

Comment:
nit: undo this part.

Reply:
This applies to `eglSwapBuffersWithDamageKHR()`, so I think we need to keep it as `@no_replay`

Reply:
The point was that you are needlessly swapping around the annotations. Since it's committed now, it doesn't really matter anymore.

Reply:
Oh OK, my bad! Thanks for the clarification.